content/develop/development-guides/consumer-vignettes.md

```diff
@@ -0,0 +1,13 @@
+---
+path: "/develop/development-guides/consumer-vignettes"
+date: "2018-08-14"
+title: "Consumer Vignettes"
+---
+
+## How Do I Find Examples of Developing Against DCP Services?
+
+All HCA DCP software has an associated GitHub repository that will include more detailed technical documentation of the software.   
+
+In additional, we offer resources for you to see references of how to perform certain actions programmatically when interacting with the DCP. These "consumer vignettes" walk through the experience of leveraging DCP functionality as written by a DCP implementation team member. These vignettes also give a usability status to give you a feel for how inituitive the walk through was for the developer; giving you a sense of how much work it may take to adapt the example to your own needs.
```

Comment:
Just a comment: 
This is more a comment on the Consumer Vignettes page than this text. I approve the text as it stands. 

My feeling is that rating the usability of the data model does little for the developer using the library except to undermine their confidence in the system. It is what it is until we improve it.  Isn't it enough to just document the tricky parts with code examples?
We should just open the channels for feedback